### ISSUE TITLE
copy from

### ISSUE THREAD

**Issue by liu-x-p** [JakobPoncelet/FHVAE_TF2#3]
When training the model, how to set copy_from in tr_dset?

**Comment by JakobPoncelet** [comment]
You could add it in scripts/train/hs_train_loaders.py when initializing tr_dset.

The copy_from is only used in the development set dt_dset now, to copy the list of labels used in the training set.
You don't really have to use it for the training set.

**Comment by liu-x-p** [comment]
> You could add it in scripts/train/hs_train_loaders.py when initializing tr_dset.
> 
> The copy_from is only used in the development set dt_dset now, to copy the list of labels used in the training set.
> You don't really have to use it for the training set.

**Comment by liu-x-p** [comment]
> > You could add it in scripts/train/hs_train_loaders.py when initializing tr_dset.
> > The copy_from is only used in the development set dt_dset now, to copy the list of labels used in the training set.
> > You don't really have to use it for the training set.

**Comment by liu-x-p** [comment]
> You could add it in scripts/train/hs_train_loaders.py when initializing tr_dset.
> 
> The copy_from is only used in the development set dt_dset now, to copy the list of labels used in the training set.
> You don't really have to use it for the training set.

Thanks.
I find you use different path for data and scripts, so I imitate that setting. But it gets errors. I have tried both absolute path & relative path, and it doesn't work.

No such file or directory: 'absolute path for /timit_np_fbank'->'.datasets/timit_np_fbank'
![2020-06-25 17-05-00屏幕 截图](https://user-images.githubusercontent.com/58974386/85692428-f61d7080-b707-11ea-8fee-df21634bc834.png)

**Comment by JakobPoncelet** [comment]
Yes, that is sloppy coding from my side, that error can happen when your data setup is different from mine. I will change it in the future.

You should make an empty 'datasets' folder at the same level of the scripts working directory (so that you have the directories 'scripts', 'fhvae', 'preprocessing_examples', etc. and 'datasets' at the same level), and in there create a symbolic link to the folder where you actually want to put the data. And then get rid of line 62-68 of scripts/train/run_hs_train.py.

**Comment by liu-x-p** [comment]
> Yes, that is sloppy coding from my side, that error can happen when your data setup is different from mine. I will change it in the future.
> 
> You should make an empty 'datasets' folder at the same level of the scripts working directory (so that you have the directories 'scripts', 'fhvae', 'preprocessing_examples', etc. and 'datasets' at the same level), and in there create a symbolic link to the folder where you actually want to put the data. And then get rid of line 62-68 of scripts/train/run_hs_train.py.

Then the question came back to 'copy_from' . I deleted all data and run it from processing timit , but still have this problem, and this assertion from seq_dataset.py, I think this means I need to give a value to 'copy_from', because it comes when 'copy_from==none' ?
![2020-06-25 21-49-25屏幕  截图](https://user-images.githubusercontent.com/58974386/85735808-9091aa00-b730-11ea-91b9-eef0db9a8ae9.png)

**Comment by JakobPoncelet** [comment]
Copy_from=None is not the error. Please print 'path' on line 100 of seq_dataset.py and make sure the file exists and is not empty. My guess is that you didn't specify the right path for your 'all_facs_phones.scp' file; or something went wrong during the preprocessing script in creating those files.

**Comment by liu-x-p** [comment]
> Copy_from=None is not the error. Please print 'path' on line 100 of seq_dataset.py and make sure the file exists and is not empty. My guess is that you didn't specify the right path for your 'all_facs_phones.scp' file; or something went wrong during the preprocessing script in creating those files.

Oh, the file exists but it is an empty file, but when preprocessing it didn't report errors. I should check it out.
Thank you! 
Happy Dragon Boat Festival 端午节快乐
